pages/login/index.js

```diff
@@ -1,8 +1,8 @@
-import styles from "./internals/styles/login.css";
-import Link from "next/link";
+import styles from "./styles/login.css";
 import { inputInfo } from "./internals/data/inputInfo";
 import LabeledInput from "./internals/components/LabeledInput";
 import { useState, useEffect } from "react";
+import { useRouter } from "next/router";
 
 export default function Login() {
   const [users, setUsers] = useState([]);
```

Comment:
**note:** It won't let me select the line below not too sure why, but look into changing the login state from an array to another type.